pages/genre/genre.go

```diff
@@ -42,3 +45,22 @@ func containsLowerCase(array []string, search string) bool {
 
 	return false
 }
+
+func averageGenreScore(user *arn.User, animes []*arn.Anime) string {
```

Comment:
Nowhere in this function is anything genre related accessed.
Functions should be named correctly and imply what they are doing.
This function simply takes 2 inputs (user, animes) and returns the average rating of the given anime that also intersect with the user's anime list. Since the list of anime is passed as a parameter, it could be anything, even non-Genre related (and this is not a bad thing).
My only complaint here is the name. It should be `averageScore` and also have a little comment about it starting with the function name just like most other functions do (VSC would warn you about that automatically).